### ISSUE TITLE
Templater use in Canvas

### ISSUE THREAD

**Issue by huffn** [4rtamis/obsidian-brumes#2]
I'm so glad I stumbled across this theme while searching Github for City of Mist repos.

I've been playing around with the Iceberg canvas, and was wondering if there was a way to tie the add card on a canvas to a template. That way it could randomly generate the rotation as well as prompt if it was a simple card or a toolkit with person or location as the image.

**Comment by 4rtamis** [comment]
Yes, that would be a great improvement for sure. I've noticed that Templater won't work at all with canvas (you can try adding a tag in a random canvas note and it will show an error message), so I'm not sure I will be able to do anything...

Overall, I'd like to simplify the process of creating an Iceberg and I was thinking about implementing a whole plugin dedicated to this task. In a distant future, you could be dragging and  dropping Iceberg cards on the layout and everything would be working instantly.

Thanks for the suggestion and, hopefully, I'll find a temporary workaround for the next release ! If you have any idea on how to do the trick, don't hesistate to submit it here, or even make a pull request to the `README.md` file.

**Comment by huffn** [comment]
Currently the only way I've found templater to work with canvas is to create a note, convert it to a file, and then templater recognizes it as a spot to execute templates.

I was playing around to see if I could add icons in the action bar on the bottom of the canvas, but I haven't found anything I could hook into yet.
